### PR TITLE
minimal change to allow db tracing

### PR DESCRIPTION
**Pull request by paulyea**
minimal change needed to be able to pass in a sql driver with contexts

### CODE REVIEW

**Review by martinxsliu — APPROVED** [review]
You may want to force push away those last two commits.

File: interface.go [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
-import "database/sql"
+import (
+	"context"
+	"database/sql"
+)
 
 // SQLCommon is the minimal database connection functionality gorm requires.  Implemented by *sql.DB.
 type SQLCommon interface {
 	Exec(query string, args ...interface{}) (sql.Result, error)
```
Comment (martinxsliu) [inline interface.go:10]: Just as a sanity check, do all the tests pass if you remove the non-context functions from this interface?

Comment (paulyea) [reply interface.go:10]: Functions such as `HasIndex` calls the non-context functions so the tests will fail. Would you prefer if I change all non-context function calls with context function call with `context.Background()`?

Comment (paulyea) [reply interface.go:10]: though that feels little unnecessary if you're going for the minimal amount of change needed for making the jaeger spans appear.

Comment (martinxsliu) [reply interface.go:10]: Oh yeah good point, no need to do it then. I was just thinking if that would be a low cost way of double checking that we didn't leave anything out.

**Pull request merged**